src/pages/docs/en/basics/local.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Developing Locally
+section: Basics
+position: 3
+hidden: false
+layout: "@docs"
+---
+
+## Starting a Development Server
+
+Once you've created a Space project, you can start developing your app locally using the Space CLI's `space dev` command.
+
+```bash
+space dev # start a local development server and open the app in your browser
+```
+
+Behind the scenes, the Space CLI is:
+
+- Generating a data key to allow your app to access the Bases and Drives of your project
+- Running each of your micro in a separate process
+- Exposing all of your micros through a single entrypoint, on `localhost:4200` by default.
+
+If you want to open the app in your browser, you can use add `--open` flag:
+
+```bash
+space dev --open # start a local development server and open the app in your browser
+```
+
+You can customize the development command for each micro in your `Spacefile` using the `dev` key.
+
+```yaml
+v: 0
+micros:
+  - name: client
+    src: ./client
+    engine: svelte
+    primary: true
+    path: client
+  - name: api
+    src: ./api
+    engine: nodejs16
+    dev: nodemon index.js # start the micro in development mode
+    path: api
+```
+
+The specified `dev` commands need to start a web server that listens on the port specified with the `PORT` environment variable. This way the CLI can properly forward requests to the micro.
+
+You can find guide specific to each micro engine in the Quickstart Guides.
+
+## Advanced Usage
+
+If you want to have more control over the development process, you can also split the `dev` command into two parts:
+
+- First, start micros individually using the `space dev up` command
+- Then, start the entrypoint server using the `space dev proxy`
+
+For our example above, this would look like this:
+
+```bash
+# Start the micros individually
+space dev up client
+space dev up api
+
+# Start the entrypoint server
+space dev proxy
+```
+
+Some use cases for this approach include:
+
+- Running each micro in a separate terminal window
+- integrating micros with other development tools (ex: vscode tasks)
+
+The `space dev` command is just a shortcut for the combination of `space dev up` and `space dev proxy`.
+
+## Running scheduled actions manually
+
+You can trigger [scheduled actions](/docs/en/basics/micros#scheduled-actions) from your local development server by using the `space dev trigger` command.
+
+First make sure you have defined a scheduled action in your Spacefile and set up the right handler in your Micro:
+
+```yaml
```

Comment:
Just a thought: would be cool if we have a component for the docs where you can show different code blocks as "file" tabs. Then we could have a tab for the Spacefile and one for the Micro handler